src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="http://localhost:8080/"
+    method="post"
+    >
+      <div class="box">
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="form-field">
+            <label for="surname">Surname:</label>
+            <input
+            type="text"
+            name="surname"
+            id="surname"
+            >
+          </div>
+          <div class="form-field">
+            <label for="name">Name:</label>
+            <input
+            type="text"
+            name="name"
+            id="name"
+            >
+          </div>
+          <div class="form-field">
+            <label for="age">How old are You?</label>
+            <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
```

Comment:
make sure to fix [checklist p.2](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) across all your document